src/components/OtherConditions/OtherConditions.tsx

```diff
@@ -1,28 +1,71 @@
-import React from "react";
-import styles from "./OtherConditions.module.css";
 import { conditions } from "../../api/api";
-import {Row, Col} from 'antd';
+import { Row, Col } from "antd";
+import { ConditionsStyle } from "./style";
+import { Conditions } from "./OtherConditions.type";
+import { useQuery } from "react-query";
+import { getWeather } from "../../api/real-api";
+import loader from "../../assets/infinite-spinner.svg";
 
 const OtherConditions = () => {
+  const { data, isLoading, error, isError } = useQuery("weather", () => getWeather('New York'));
+  
+
+  if (isLoading)
+    return <img src={loader} alt="Loading..." width={50} height={50} />;
```

Comment:
I already saw this code. Consider moving it to reusable component to avoid duplication